src/features/search/pages/Search/SearchN1/SearchN1.native.test.tsx

```diff
@@ -20,9 +21,7 @@ jest.mock('libs/firebase/analytics/analytics')
 jest.mock('features/navigation/TabBar/routes')
 jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(false)
 
-jest
-  .spyOn(useGTLPlaylists, 'useGTLPlaylists')
-  .mockReturnValue({ isLoading: false, gtlPlaylists: gtlPlaylistAlgoliaSnapshot })
+const mockUseGtlPlaylist = jest.spyOn(useGTLPlaylists, 'useGTLPlaylists')
```

Comment:
```suggestion
const mockUseGtlPlaylist = jest.spyOn(useGTLPlaylists, 'useGTLPlaylists')
mockUseGtlPlaylist.mockReturnValue({** isLoading: false, gtlPlaylists: gtlPlaylistAlgoliaSnapshot })
```
Pourquoi ne pas laisser l'instruction avec `mockReturnValue` vu qu'on l'utilise partout avec les même paramètres ?
Pourquoi est-ce que t'as plutôt besoin de `mockReturnValueOnce` ?

Reply:
pour pouvoir tester le FF j'ai besoin que le hook n'ai pas une valeur de retour mocké